test/controllers/sections/items/ItemsPackagingIndexControllerSpec.scala

```diff
@@ -51,11 +51,12 @@ class ItemsPackagingIndexControllerSpec extends SpecBase
 
     "when ItemsSection.isCompleted" - {
```

Comment:
This should be this now, I think? 
```suggestion
    "when ItemsSectionItems.isCompleted" - {
```